docs/deployment/linux.md

```diff
@@ -0,0 +1,125 @@
+
+Enter the directory using `cd folder_name`.
+
+## Downloading Velocity
+
+You can find the latest version [on the downloads site](https://www.velocitypowered.com/downloads). Download the latest version to your newly created folder (or use FTP to upload it).
+
+As an example, I'll use this command:
+```shell script
+wget -O proxy.jar https://ci.velocitypowered.com/job/velocity/246/artifact/proxy/build/libs/velocity-proxy-1.0.11-SNAPSHOT-all.jar
+```
+
+_Please note, this is most likely not the latest version when you're reading this. Copy the latest download link from the downloads site._
+
+## Running Velocity
+
+### The Script
+
+To easily start and stop Velocity, we will use a simple shell script.
+
+To start creating this script, use `nano start.sh`, and paste this in:
+
+```sh
+#!/bin/bash
+PROXY_JAR=proxy.jar
+RAM=512M
+
+while true
+do
+    java -XX:+UseG1GC -XX:G1HeapRegionSize=4M -XX:+UnlockExperimentalVMOptions -XX:+ParallelRefProcEnabled -XX:+AlwaysPreTouch -Xmx${RAM} -Xms${RAM} -jar ${PROXY_JAR}
+    echo "Velocity exited."
+    echo "Starting in 3... (Press [CTRL+C] to stop)"
+    sleep 1
+    echo "Starting in 2... (Press [CTRL+C] to stop)"
+    sleep 1
+    echo "Starting in 1... (Press [CTRL+C] to stop)"
+    sleep 1
+done
+```
+
+_Note: This uses Tux's recommended HotSpot Java flags as of 04/25/2020._
+
+When you're done, press <kbd>CTRL+S</kbd> and <kbd>CTRL+X</kbd>. Your start script has now been created, and running `ls` should now show `start.sh`.
+
+To actually _run_ this script, you first will need to add executable permissions to the file.
+
+To do this, run:
+
+```shell script
+chmod +x start.sh
+```
+
+Now that the script is executable, we can run the proxy! Using the command:
+
+```
+./start.sh
+```
+
+will start up the Velocity jar. In the **rare** event that the proxy crashes, it will be automatically turned back on. If you wish to stop the proxy, pressing CTRL+C once will stop Velocity, and pressing CTRL+C again while it's not running will end the script.
+
+For now, end the script. We have one last thing to do.
+
+### Running Velocity in the Background
+
+As mentioned earlier, this is most likely a case where you'll want to run Velocity in the background.
+
+To do this, run this command:
+
+```shell script
+screen -S velocity ./start.sh
+```
+
+This will create a new `screen`, which will automatically run your Velocity script. Now that it's up and running, you can press <kbd>CTRL+A+D</kbd> to exit the screen, and now you can go about your day :)
+
+# Extra Steps
+
+## Configuring Velocity
+
+It's great that Velocity is running, but now you need servers to connect to! Refer to the [Configuring Velocity guide found here](https://newsite-staging.velocitypowered.com/wiki/users/configuration/).
+
+## Firewall Access
+
+In some cases, your Linux server may be configured to block access on Velocity's running port. Each Linux distribution has its own way of configuring firewall access, which won't be covered here.
```

Comment:
Add a link to the DigitalOcean which includes instructions for most distros

Reply:
I can only find [this guide](https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-with-ufw-on-ubuntu-18-04) which provides instructions for Debian 9/10, and the last few Ubuntu LTS builds. Is this what you were referring to?

Reply:
You're right, I recall finding a guide including iptables, ufw and firewalled all in the same place but can't find it 😕

We could add your link above + https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-using-firewalld-on-centos-7. Afaik that's what most MC servers run on.

Reply:
* `firewalld` (Fedora derivatives): https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-using-firewalld-on-centos-7
* `ufw` (Ubuntu): https://www.digitalocean.com/community/tutorials/how-to-set-up-a-firewall-with-ufw-on-ubuntu-18-04

Note that iptables is considered legacy software and people are urged to move to nftables, these are based on nftables.

Reply:
I added links for each version provided in these guides so it's clear which link goes to which version.